rust/migrate-wicked/tests/test.sh

```diff
@@ -45,7 +47,7 @@ for test_dir in ${TEST_DIRS}; do
     $MIGRATE_WICKED_BIN show $test_dir/wicked_xml
     if [ $? -ne 0 ] && [ "$expect_fail" = false ]; then
         error_msg ${test_dir} "show failed"
-        RESULT=1
+        failed=true
```

Comment:
Not related to your change, but maybe we should stop the test here and continue with the next one?

Reply:
Well if something is broken in just the show code I think it's still good to test the migration code. Makes it easier to identify where a issue is present.

Reply:
Or we  do it like:
```suggestion
        FAILED_TESTCASES+=("${test_dir}::show")
```

And
```
FAILED_TESTCASES+=("${test_dir}::migrate")
# or
FAILED_TESTCASES+=("${test_dir}::compare_config")
```
for the others...
WDYT?